### PR TITLE
Harvey magana

### PR DESCRIPTION
**Pull request by harvey-magana**
Plants schema updated so that the team can use something to manage the date of when to water the plants and I added the right data type for image upload. I also updated the put method in the plants router.

### CODE REVIEW

**Comment by harvey-magana** [comment]
I'm leaving this pull request as is. To resolve the conflict would also merge it, so all you would need to do is to add 
-           tbl.string('image_url');
-           tbl.date('isWatered');
 to the main plant portion of the schema. I was going to do this but it would commit and merge if I did that. I will leave this to you so you have an opportunity to review it before merging it.

**Review by evllz — CHANGES_REQUESTED** [review]
"async" is not used on "put" route but "await" is used inside. Please add "async" after "restricted" middleware

**Review by evllz — APPROVED** [review]

**Pull request merged**
